src/libs/NextStepUtils.ts

```diff
@@ -76,9 +91,10 @@ function buildNextStep(report: OnyxEntry<Report>, predictedNextStatus: ValueOf<t
     const policy = allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${policyID}`] ?? ({} as Policy);
     const {harvesting, autoReportingOffset} = policy;
     const autoReportingFrequency = PolicyUtils.getCorrectedAutoReportingFrequency(policy);
-    const submitToAccountID = PolicyUtils.getSubmitToAccountID(policy, ownerAccountID);
+    const submitToAccountID = ReportUtils.getSubmitToAccountID(policy, ownerAccountID);
```

Comment:
Sorryyyyy I commented elsewhere that I think this one should actually remain in `PolicyUtils` 😬

Reply:
np! resolved